packages/fxa-payments-server/src/components/PaymentForm/index.test.tsx

```diff
@@ -227,79 +227,79 @@ describe('Legal', () => {
     it('renders Localized for daily plan with correct props and displays correct default string', async () => {
       const plan_id = 'plan_daily';
       const plan = findMockPlan(plan_id);
-      const expectedMsgId = 'payment-confirm-day';
+      const expectedMsgId = 'payment-confirm-with-legal-links-day';
       const expectedMsg =
-        'I authorize Mozilla, maker of Firefox products, to charge my payment method <strong>$5.00 daily</strong>, according to payment terms, until I cancel my subscription.';
+        'I authorize Mozilla, maker of Firefox products, to charge my payment method <strong>$5.00 daily</strong>, according to <termsOfServiceLink>Terms of Service</termsOfServiceLink> and <privacyNoticeLink>Privacy Notice</privacyNoticeLink>, until I cancel my subscription.';
```

Comment:
Might be nice to DRY up some of these strings. But, when I thought about doing it, it looked like the process would basically be duplicating the implementation of the function under test